Samples/JavaDependencySampleApp/Package.swift

```diff
@@ -99,6 +100,25 @@ let package = Package(
       ]
     ),
 
+    .target(
+      name: "JavaJson",
```

Comment:
Can we call this "OrgAndrejsJson", so it follows the recently established pattern and is more clear that it's wrapping java?
```suggestion
      name: "OrgAndrejsJson",
```